cli/tests/pcluster/templates/test_cluster_stack.py

```diff
@@ -70,6 +70,25 @@ def test_cluster_builder_from_configuration_file(
     _generate_template(cluster, capsys)
 
 
+def test_no_security_groups_created_from_configuration_file(mocker, capsys, pcluster_config_reader, test_datadir):
```

Comment:
since you refactored the code can you please create a test to verify the mount targets (and the security groups) are created when using not existing EFS?

Reply:
This should be fully covered by integration tests already

Reply:
if so, it's better to remove the end-to-end test and write instead a unit test, it's faster and cheaper